tensorflow/python/keras/applications/efficientnet_weight_update_util.py

```diff
@@ -0,0 +1,307 @@
+"""Utils for EfficientNet models for Keras.
+Write weights from  ckpt file as in original repo
+(https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet)
+to h5 file for keras implementation of the models.
```

Comment:
Please add a sample usage for this script, so that ppl can easily copy paste to start with.